be/CMakeLists.txt

```diff
@@ -769,9 +769,9 @@ add_subdirectory(${SRC_DIR}/runtime)
 add_subdirectory(${SRC_DIR}/service)
 add_subdirectory(${SRC_DIR}/udf)
 
-if (${BUILD_META_TOOL} STREQUAL "ON")
+#if (${BUILD_META_TOOL} STREQUAL "ON")
     add_subdirectory(${SRC_DIR}/tools)
-endif()
+#endif()
```

Comment:
This modification may be unnecessary due to the merge #7948 .

Reply:
yes, i will change the code.